<launch>
  <node name="goal_provider_node" pkg="goal_provider" type="goal_provider_node" output="screen">
    <param name="callbacks_till_goal_update" value="1000"/>
    <param name="competition_length" value="10" />
    <param name="time_cutoff" value="8" />
    <param name="goal_attempt_limit" value="5" />
    <param name="testing" value="true" />
    <param name="path" value="/home/enda/fuerte_workspace/sandbox/" />
  </node>
</launch>
